### PR TITLE
Prepare for Python 3.11

### PR DESCRIPTION
**Pull request by nafiz1001**
https://206.12.92.46/issues/1192

> Write in the DEPLOYMENT.md for the version 4.4 (last section) the operational step to replace Python3.8 by 3.11 in the actual release (replacing the env and posible other required manual steps). Also update the README.md and other doc to reflect the updated python version.

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -1,90 +1,92 @@
...
-distlib==0.3.0
-distro==1.4.0
-Django==3.1
+coverage==7.2.7
+cryptography==41.0.1
+defusedxml==0.7.1
+diff-match-patch==20230430
+distlib==0.3.6
+distro==1.8.0
+Django==4.2.2
```
Comment (nafiz1001) [inline backend/requirements.txt:21]: Ooolala ;)

Comment (UlysseFG) [reply backend/requirements.txt:21]: yeah the Django upgrade might come with some changes ... The warnings you are seeing are likely a new requirement that they are phasing in ...

**Comment by UlysseFG** [comment]
Check with IT for the procedure to perform to update the python on the servers (dev, qc, prod) to update the DEPLOYMENT.md file accordingly. Update also the README.md of the backend to make sure we use the correct python and pip version (3.11 instead of 3.8) to generate the environment. Once it is done we can deploy the upgrade to DEV by merging to master and regenerating the environment on dev (not sure if pip is still upgraded or if it was rolled back).

**Review by UlysseFG — APPROVED** [review]

**Comment by UlysseFG** [comment]
Let me know when you merge i'll poke Raul and try to regenerate the Env with the new Python on dev ...

**Pull request merged**